src/core/server/context/container/context.mock.ts

```diff
@@ -12,6 +12,7 @@ export type ContextContainerMock = jest.Mocked<IContextContainer<any>>;
 
 const createContextMock = (mockContext = {}) => {
   const contextMock: ContextContainerMock = {
+    // @ts-expect-error tsc cannot infer ContextName and uses never
```

Comment:
@elastic/kibana-core I didn't manage to work around this. We can fix the problem later: this PR blocks migration to TS projects.